src/services/asset-inventory/cloud-service/cloud-service-detail/modules/cloud-service-history-detail/CloudServiceHistoryChangesTab.vue

```diff
@@ -21,11 +21,12 @@
                     <template #header>
                         <div class="cloud-service-history-changes-code-header-wrapper">
                             <div>
-                                <span>data</span>
-                                <p-i v-if="selectedKeyMenu" name="ic_arrow_right" color="#898995"
-                                     scale="0.7"
-                                />
-                                <span>{{ selectedKeyMenu }}</span>
+                                <span v-for="(key, index) in selectedKeyMenu.split('.')" :key="index">
+                                    <p-i v-if="selectedKeyMenu && index !== 0" name="ic_arrow_right" color="#898995"
```

Comment:
pIcon에 컬러를 주입하려면, 아래와 같이 `inherit`으로 준 뒤, 상위 객체 컬러를 따르게 해야 합니다! (이유가 가물한데 아마 svg라 그런 걸거예요.)
따라서 PIcon을 div나 span으로 감싼 뒤, 해당 엘레멘트에 color값을 주어야 합니당

```suggestion
                                    <p-i v-if="selectedKeyMenu && index !== 0" name="ic_arrow_right" color="inherit"
```

Reply:
앗 그리고 컬러코드는 반드시!!!! 저희 컬러셋 이용해주세요!